<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableForm</class>
 <widget class="QWidget" name="TableForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string> tableView_demo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>  删除数据</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>  修改表头背景</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="6">
        <widget class="QTableView" name="tableView"/>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string> 修改表格样式</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>复制数据 </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>  新增数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>tableWidget_demo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="1">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>自动调节行高</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>表格可编辑</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="checkBox_5">
         <property name="text">
          <string>隐藏行表头</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>  删除一行数据 </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>  修改列表头样式 </string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string> 删除一列数据</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>  新增一列数据 </string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string>  修改行表头样式 </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="13">
        <widget class="QTableWidget" name="tableWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>5</number>
         </property>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string>清除表数据</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string> 新增一行数据</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>自动调节列宽</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string>隐藏列表头</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string> 位置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>居中</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>靠左</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>靠右</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>行高设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>列宽设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string> 导出数据ecxel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
